.github/dependabot.yml

```diff
@@ -4,3 +4,8 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
+    groups:
+      dependencies:
```

Comment:
This repo doesn't have updates enabled for GitHub Actions - I think they should be enabled too. At which point, I think it might be worth naming the group something like `java-dependencies` to differentiate them?

(IMO we don't need to enable grouping for GitHub Actions - we haven't for other repos, since on the most part the actions use the major version tags anyway, so there would be no minor/patch versions to group)

Reply:
added GH action updates and renamed this group to `java-dependencies`